.github/workflows/run-tests.yml

```diff
@@ -1,7 +1,13 @@
 name: run-tests
 
 on:
-  - push
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
I thought this might be good as the actions are free anyway. If Xero pushes out a new package update, we'll automatically test against it.

Happy to remove tho.